### ISSUE TITLE
Flood does not honour default download directory

### ISSUE THREAD

**Issue by awsms** [jesec/flood#531]
Whenever I upload/download something in a particular directory, my next upload/download will be set in the said directory.
If I remove the path, then click add torrent, it will still download the torrent to the previously used folder.

**Comment by disconsolated** [comment]
Any update on this? Facing the same issue currently.

**Comment by jesec** [comment]
This is expected behavior.

Flood prioritizes the last used destination of a given tag over the default destination of the torrent client. It does not have a strong concept of the default download destination. Flood will suggest the destination it is going to use by pre-filling the destination textbox.

Destinations should be specified explicitly if you wish to use an explicit destination.

**Comment by awsms** [comment]
What is the point of having a default folder option then?

**Comment by dajabe** [comment]
Yeah I've been having this problem when dragging and dropping torrents into the app. It will always go to /config/download instead of /download. Pretty frustrating when it's setup as a docker and one of those has lots of space and the other doesn't.

**Comment by awsms** [comment]
I don't even get the logic behind this: why would the path I use to make a torrent become the new "default folder" for downloads?
I'm back to ruTorrent for making torrents because of this ridiculous design.

**Comment by scottshanafelt** [comment]
Experienced the same problem and came looking for a solution.  I really want to switch to this client but this issue may prevent me from doing so.

**Comment by trim21** [comment]
I think this is a very bad idea, make default download dir completely meanless

**Comment by trim21** [comment]
if anyone doesn't like this behavoir, send a PR and I'll merge it.
